### PR TITLE
Latest tasks

### PR DESCRIPTION
**Pull request by Ikiselev7**
just latest tasks

### CODE REVIEW

**Comment by aabarmin** [comment]
There are a lot of style issues, could you please fix them?

**Comment by Ikiselev7** [comment]
Yes, they are in the project

File: src/main/java/com/epam/university/java/core/task028/Task028Impl.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import java.util.stream.Stream;
+
+/**
+ * Created by ilya on 03.10.17.
+ */
+public class Task028Impl implements Task028 {
+
+    private int varCounter;
+    private int[] powers;
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task028/Task028Impl.java:11]: May be it's better to use collections?)

Comment (Ikiselev7) [reply src/main/java/com/epam/university/java/core/task028/Task028Impl.java:11]: Yes, and you know that I can, just here I had been doing some optimization last time, and think that this is faster way, but I don't sure very well.

**Pull request merged**